lib/scope.js

```diff
@@ -1548,14 +1547,15 @@ module.exports = {
       // future we should look into what happens with a `fieldset` for
       // `choices:`, etc.
       let custom_datatype = null, matches = null;
-      let group = elem.closest( `.da-form-group` );
-      if ( group && group.className ) {
-        matches = ` ${ group.className } `.match(/da-field-container-datatype-([^ ]+) /);
-      }
-      if ( matches ) {
-        custom_datatype = matches[1];
+      if (elem) {
+        let group = elem.closest( `.da-form-group` );
+        if ( group && group.className ) {
+          matches = ` ${ group.className } `.match(/da-field-container-datatype-([^ ]+) /);
+        }
+        if ( matches ) {
+          custom_datatype = matches[1];
+        }
```

Comment:
What should happen if the `elem` doesn't exist? Error? Warning? When is this called?